components/reports/reports-filters.tsx

```diff
@@ -1,23 +1,25 @@
-import { Filter } from "lucide-react";
```

Comment:
needs 'use client' for `Drawer`

Reply:
Added in [2f95c15](https://github.com/VoiceDeck/app/pull/47/commits/2f95c157237616fad12805ce3ab1b057c39bf83d)+